andela/mrm-mobile
# 159728988 Filter rooms by capacity

### What does this PR do?
Allows the user to filter rooms by Capacity.

### Description of Task to be completed.
[x] Allow users to select filter by capacity.
[x] Filter all rooms by capacity.
[x] Make Filtering by capacity responsive.
[  ] Write tests for the functionality.

### How should this be manually tested?
1. Set up the project locally as shown in README.
2. Log in to Converge.
3. Navigate to Rooms (Location/Office/Block/Floor/Room).
4. In room, click find room.
5. Check the capacity filter.

### What are the relevant pivotal tracker stories?
[A user can find a room by using the filter  capacity](https://www.pivotaltracker.com/story/show/159728988)
[159728988](https://www.pivotaltracker.com/story/show/159728988)

### Screenshots
![ezgif com-optimize](https://user-images.githubusercontent.com/33759757/44350826-f9f76080-a4a8-11e8-90f6-4b3bc44a656c.gif)
